temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
+print("hello")
```

Comment:
Avoid using print statements for logging. Consider using a logging framework to provide better control and configurability over log messages.